_posts/2023-02-16-SheltonBlackwell-proportional-share-meme.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Fairness Rising
+author: Shelly
+image: fairness_stonks.jpg
+chapter: 9 
+---
+The fairness metric is simply the time the first job completes divided 
+by the time that the second job completes. It ranges from .0 to 1, with 0 being the least fair and 1 being the most fair. The jobs that finish within the same time will have a higher fairness. 
```

Comment:
It is true that more time would grant each process more time slices and that would increase fairness, but really only with the lottery scheduler. Can you mention that your meme is only about the lottery?

Reply:
This part of your explanation is still off: "The jobs that finish within the same time will have a higher fairness." Your meme has this correct.